content/docs/searching.md

```diff
@@ -523,9 +523,9 @@ Anchors match a zero-length position in the line, rather than a particular chara
 
 * `\B` ⇒ Not a word boundary. It represents any location between two word characters or between two non-word characters.
 
-* `\A` or `\'` ⇒ The start of the matching string.
+* `\A` or `` \` `` ⇒ The start of the matching string.
```

Comment:
Could you update per [Alan's comments](https://community.notepad-plus-plus.org/post/65053): "The start of the file"